src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/Prisoner.kt

```diff
@@ -38,12 +40,24 @@ open class Prisoner {
   @Schema(description = "Book Number", example = "38412A")
   var bookNumber: String? = null
 
+  @MultiField(
+    mainField = Field(type = FieldType.Text),
+    otherFields = [
+      InnerField(type = FieldType.Keyword, suffix = "keyword")
+    ]
+  )
   @Schema(required = true, description = "First Name", example = "Robert")
```

Comment:
will be indexed twice as text and keyword. The keyword field will be call `firstName.keyword`